blaze/compute/pandas.py

```diff
@@ -76,6 +76,21 @@ def compute_up(t, s, **kwargs):
     return compute_up(t, s.to_frame(), **kwargs)
 
 
+@dispatch(Interp, Series)
+def compute_up(t, data, **kwargs):
+    if isinstance(t.lhs, Expr):
+        return pd.Series(data.values % t.rhs)
+    else:
+        return pd.Series(t.lhs % data.values)
+
+
+@compute_up.register(Interp, Series, (Series, base))
+@compute_up.register(Interp, base, Series)
+def compute_up_pd_interp(t, lhs, rhs, **kwargs):
+    return pd.Series(lhs.values % rhs.values)
```

Comment:
If this isn't raising then I suspect you don't have coverage for all the dispatch cases listed here.